Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m

```diff
@@ -72,6 +80,10 @@ - (id)init
     [self setTabBarItem:tabBarItem];
     
     [self setEdgesForExtendedLayout:UIRectEdgeTop];
+#if AD_MAM_SDK_TESTING
+    [[IntuneMAMComplianceManager instance] setDelegate:self];
```

Comment:
does MAM SDK holds a weak reference to its delegate?

Reply:
yep